react-native-reanimated.d.ts

```diff
@@ -58,7 +58,7 @@ declare module 'react-native-reanimated' {
     export type Adaptable<T> =
       | T
       | AnimatedNode<T>
-      | ReadonlyArray<Adaptable<T>>;
+      | ReadonlyArray<T | AnimatedNode<T> | T[] | AnimatedNode<T>[]>;
```

Comment:
```suggestion
      | ReadonlyArray<T | AnimatedNode<T> | ReadonlyArray<T> | ReadonlyArray<AnimatedNode<T>>>;
```

Reply:
Actually, this is probably more accurate:

```suggestion
      | ReadonlyArray<T | AnimatedNode<T> | ReadonlyArray<T | AnimatedNode<T>>>;
```